test/test_files/ctv_godunov_plm/ctv_godunov_plm.i

```diff
@@ -0,0 +1,47 @@
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#            SIMULATION STOP            #
+#.......................................#
+time.stop_time               =   0.2   # Max (simulated) time to evolve
+time.max_step                =   200   # Max number of time steps
+
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#         TIME STEP COMPUTATION         #
+#.......................................#
+time.fixed_dt         =   -0.005        # Use this constant dt if > 0
+time.cfl              =   0.5        # CFL factor
+
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#            INPUT AND OUTPUT           #
+#.......................................#
+time.plot_interval  =  20   # Steps between plot files
+time.checkpoint_interval =   -1  # Steps between checkpoint files
+io.KE_int = 1        # calculate kinetic energy 
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#               PHYSICS                 #
+#.......................................#
+incflo.gravity          = 0.  0.  0.  # Gravitational force (3D)
+incflo.density             = 1.          # Reference density 
+incflo.use_godunov      = 1
+incflo.use_ppm = 0
```

Comment:
So plm has not received as much testing in amr-wind as ppm. @michaeljbrazell  thoughts?

Reply:
I had the same thought... I would try ppm first.

Reply:
I've done PLM for MMS and CTV and it seems fine. Maybe it's nice to test PLM too.